pkgs/desktops/gnome-3/apps/gnome-weather/default.nix

```diff
@@ -1,29 +1,24 @@
 { stdenv, fetchurl, pkgconfig, gnome3, gtk3, wrapGAppsHook, gjs, gobject-introspection
-, libgweather, intltool, itstool, geoclue2, gnome-desktop }:
+, libgweather, meson, ninja, geoclue2, gnome-desktop, python3 }:
 
 stdenv.mkDerivation rec {
   name = "gnome-weather-${version}";
-  version = "3.26.0";
+  version = "3.32.0";
 
   src = fetchurl {
     url = "mirror://gnome/sources/gnome-weather/${stdenv.lib.versions.majorMinor version}/${name}.tar.xz";
-    sha256 = "965cc0d1b4d4e53c06d494db96f0b124d232af5c0e731ca900edd10f77a74c78";
+    sha256 = "0b7cqd3wfrgm0hps0cb8vhjz0bpjw955hbc4r82l626g8l3hf86w";
   };
 
-  nativeBuildInputs = [ pkgconfig intltool itstool wrapGAppsHook ];
+  nativeBuildInputs = [ pkgconfig meson ninja wrapGAppsHook python3 ];
   buildInputs = [
     gtk3 gjs gobject-introspection gnome-desktop
     libgweather gnome3.adwaita-icon-theme geoclue2 gnome3.gsettings-desktop-schemas
   ];
 
-  # The .service file isn't wrapped with the correct environment
-  # so misses GIR files when started. By re-pointing from the gjs
-  # entry point to the wrapped binary we get back to a wrapped
-  # binary.
-  preConfigure = ''
-    substituteInPlace "data/org.gnome.Weather.Application.service.in" \
```

Comment:
This service was just renamed to `org.gnome.Weather`. Do we really not need it any more? I do not see any related changes.

Reply:
Yeah, you're right, it fails to launch from .desktop file now.